pkg/eosclient/eosgrpc/eosgrpc.go

```diff
@@ -1629,6 +1631,10 @@ func (c *Client) grpcMDResponseToFileInfo(ctx context.Context, st *erpc.MDRespon
 			fi.Attrs[strings.TrimPrefix(k, "user.")] = string(v)
 		}
 
+		if fi.Attrs["sys.acl"] != "" {
```

Comment:
Duplicate of the code bellow?

Reply:
Different if statement. I can also not really pull it out because `fi.Attrs` must be populated first, which happens differently in each branch